### ISSUE TITLE
train detail

### ISSUE THREAD

**Issue by Xue9901** [google/stereo-magnification#27]
Hello!thank you for the wonderfull work!  I'm very interested in your project ! I'm training this model for about one day and one night on GPU, but the loss seems to be still sometimes very big and sometimes very small ,and how long did you take for the loss to converge during the training?  
      ps. i  set the batchsize to 8,with learning rate= 0.0002,β1 = 0.9,β2 = 0.999
